pkgs/os-specific/linux/lvm2/default.nix

```diff
@@ -53,6 +53,9 @@ stdenv.mkDerivation rec {
     sed -i /DEFAULT_PROFILE_DIR/d conf/Makefile.in
     substituteInPlace scripts/lvm2_activation_generator_systemd_red_hat.c \
       --replace /usr/bin/udevadm /run/current-system/systemd/bin/udevadm
+    # https://github.com/lvmteam/lvm2/issues/36
+    substituteInPlace udev/69-dm-lvm-metad.rules.in \
+      --replace "(BINDIR)/systemd-run" /run/current-system/systemd/bin/systemd-run
```

Comment:
Is there no `$` before `(BINDIR)`?

Reply:
no this is correct

Reply:
https://github.com/lvmteam/lvm2/blob/master/udev/69-dm-lvm-metad.rules.in#L111

Reply:
No, that's some custom substitution mechanism - see https://github.com/lvmteam/lvm2/blob/master/udev/69-dm-lvm-metad.rules.in#L111 linked in the issue referenced above.